src/flows/pipes/Notification/ExportTask.tsx

```diff
@@ -71,6 +72,11 @@ const ExportTask: FC = () => {
       return acc
     }, {})
 
+    if (!data.measurement) {
+      throw new Error('please select a measurement.')
+    }
+    const measurement = `(r) => r["_measurement"] == "${data.measurement}"`
+
```

Comment:
This is the deadman alert task, in notebooks.
I could not figure out how in the UI to trigger this type of alert --> to ensure it still works.